x/commitment/keeper/msg_server_claim_airdrop.go

```diff
@@ -0,0 +1,81 @@
+package keeper
+
+import (
+	"context"
+
+	"cosmossdk.io/math"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/elys-network/elys/x/commitment/types"
+	ptypes "github.com/elys-network/elys/x/parameter/types"
+)
+
+var AirdropWallet = "elys1wk7jwkqt2h9cnpkst85j9n454e4y8znlgk842n"
```

Comment:
This is literal string. Can be made const

Reply:
this is done for running tests, test suite uses cosmos... based address